buildspec_publish_dockerhub.yml

```diff
@@ -0,0 +1,32 @@
+version: 0.2         
+phases:
+  install:
+    runtime-versions:
+      golang: 1.12
+  pre_build:
+    commands:
+      - echo Publish the image to DockerHub
+  build:
+    commands:
+      # Pull the image that we built and pushed in the `Build` stage
+      - 'ecs-cli pull amazon/aws-for-fluent-bit:latest'
+                                                               
+      # When we pull an image from ECR, it comes with accountId, ECR registry, and region name.
+      # Tag the image only with the name.
+      # Don't forget to update your account and region in the image registry.
+      - 'docker tag ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/amazon/aws-for-fluent-bit:latest amazon/aws-for-fluent-bit:latest'
+
+      # List the docker images 
+      - docker images
+
+      # DRY RUN: Push the image to dockerhub
+      - './scripts/publish.sh cicd-publish dockerhub'
```

Comment:
dry run isn't really needed in CI/CD

The point of dry run is for a human to be able to check that they have permission to log into docker hub.

Reply:
I was not sure. Thanks. Will remove this step.